client/src/Clusters/TopCharts/SingleAppContainer.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+ 
+class SingleAppContainer extends Component {
+  render() {
+    return (
+      <div>
+        <p class="num"> {this.props.listNum} </p>
+        <img
+          id="app_icon_img"
+          class="img-fluid"
+          alt="icon 1"
+          src={this.props.url}
+        />
+        <div class="app_details">
+          <h6 class="app_title"> {this.props.title} </h6>
+          <p class="grey_text"> {this.props.category} - Category </p>
+          <p class="grey_text">
+            {this.props.rating}
+            <img
+              id="rating_star_icon"
+              width="12px"
+              alt="rating star"
+              src="https://www.pngkit.com/png/detail/7-78706_free-png-grey-star-png-images-transparent-grey.png"
```

Comment:
When using an image, you should save it locally (such as this repo directly or some kind of storage). if the third party website goes down, then this image link won't work either.

But in this case, I would recommend using SVGs or font icons. They are easier to scale up without loser quality and easier to colorize as well. This image also has a white background, which will show on anything other than a white page. When it's absolutely necessary to use PNGs, make sure the background is transparent.

For this project, most of the iconry should be using Material icons. https://material.io/resources/icons/?icon=star&style=baseline